fonduer/parser/parser.py

```diff
@@ -355,9 +355,12 @@ def _parse_sentence(self, paragraph, node, state):
         field = state["paragraph"]["field"]
         # Lingual Parse
         document = state["document"]
-        for parts in self.lingual_parse(document, text):
-            (_, _, _, char_end) = split_stable_id(parts["stable_id"])
+        sentence_list = []
+        parts_list = [x for x in self.lingual_parse(document, text)]
+        # for parts in self.lingual_parse(document, text):
+        for parts in parts_list:
             parts["document"] = document
+            # NOTE: Why do we overwrite this from the spacy parse?
             parts["position"] = state["sentence"]["idx"]
```

Comment:
We're tracking the global counter for sentences. Is the value from spacy parse reliable enough for this?